test/unit/sql/select.test.js

```diff
@@ -371,6 +371,49 @@ suite(Support.getTestDialectTeaser('SQL'), function() {
         default: 'SELECT [User].[name], [User].[age], [Posts].[id] AS [Posts.id], [Posts].[title] AS [Posts.title] FROM [User] AS [User] LEFT OUTER JOIN [Post] AS [Posts] ON [User].[id] = [Posts].[user_id];'
       });
     });
+
+    it('include (subQuery alias)', function () {
+      var User = Support.sequelize.define('User', {
```

Comment:
sorry, didn't see the target. Please answer jan's question:

> Does this also happen on the master branch? In that case we will need a PR to master, before it can be backported to v3